### PR TITLE
Fix/21994 liveness probe

### PR DESCRIPTION
**Pull request by Wilderone**
This MR resolve two problems:
1. Remove "exec" argument for sh in livenessProbe, because of this leads to false-success of probe check
2. Move livenessProbe's code to values, so client can redefine it.

Related issue https://github.com/apache/airflow/issues/21994

I am not sure about version of this change, so just upgrade patch version.

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 ---
 apiVersion: v2
 name: airflow
-version: 1.5.0-dev
+version: 1.5.1-dev
```
Comment (potiuk) [inline chart/Chart.yaml]: I tihnk this should not be changed. We are voting on 1.5.0 and my bet is that we will issue and rc2

**Review by potiuk — APPROVED** [review]
I like this but I need @ephraimbuddy @jedcunningham @kaxil chiming in :)

**Review by ephraimbuddy — APPROVED** [review]
LGTM.

**Comment by potiuk** [comment]
The mypy static checks are unrelated (and already fixed in `main` - can you please rebase and fix the other static check (which is related :)). Pre-commit might help you with automatically fixing it

**Comment by potiuk** [comment]
(and removal of the version change too :))

File: chart/values.yaml [hunk 2]
```diff
@@ -563,6 +563,24 @@ scheduler:
     timeoutSeconds: 20
     failureThreshold: 5
     periodSeconds: 60
+    exec:
+      command:
+        - sh
+        - -c
+        - |
+          CONNECTION_CHECK_MAX_COUNT=0 /entrypoint python -Wignore -c "
```
Comment (jedcunningham) [inline chart/values.yaml]:
```suggestion
          CONNECTION_CHECK_MAX_COUNT=0 exec /entrypoint python -Wignore -c "
```

This is where the exec should be, and I think we want to keep it so `timeoutSeconds` can actually function.

File: chart/values.schema.json [hunk 3]
```diff
@@ -1504,6 +1504,20 @@
                             "description": "How often (in seconds) to perform the probe. Minimum value is 1.",
                             "type": "integer",
                             "default": 60
+                        },
+                        "exec": {
+                            "description": "Custom exec command for livenessProbe",
```
Comment (jedcunningham) [inline chart/values.schema.json]:
```suggestion
                            "description": "Command for livenessProbe",
```

I'm also not sure we need to maintain the nesting of `exec.command`. I'd rather we flatten it and just have `livenessProbe.command`. My 2c, thoughts?

File: chart/Chart.yaml (hunk 1 above)
Comment (jedcunningham) [reply chart/Chart.yaml]: Yeah, we don't need to touch this here.

**Review by jedcunningham — CHANGES_REQUESTED** [review]
Thanks for opening this! Can you also add some tests?

File: chart/values.yaml (hunk 2 above)
Comment (potiuk) [reply chart/values.yaml]: Yep.

**Review by potiuk — APPROVED** [review]

File: chart/tests/test_scheduler.py [hunk 4]
```diff
@@ -235,6 +236,9 @@ def test_livenessprobe_values_are_configurable(self):
             "spec.template.spec.containers[0].livenessProbe.failureThreshold", docs[0]
         )
         assert 444 == jmespath.search("spec.template.spec.containers[0].livenessProbe.periodSeconds", docs[0])
+        assert ["sh", "-c", "echo", "wow such test"] == jmespath.search(
```
Comment (potiuk) [inline chart/tests/test_scheduler.py:239]: :smile:

**Review by jedcunningham — APPROVED** [review]

**Comment by jedcunningham** [comment]
Thanks @Wilderone! Congrats on your first commit 🎉

**Pull request merged**